### PR TITLE
Create TODO-Core.md

### PR DESCRIPTION
**Pull request by TheSquidHQ**

### CODE REVIEW

**Comment by SparklingComet** [comment]
Looks good, but can you remove those hashtags (#) in front of every bullet point and add three new ones, followed by a space ("### ") in front of every (big) feature? Consider them as headers. Also, please change the file extension to ".md". You will see what it does

File: TODO-Core.txt [hunk 1]
```diff
@@ -12,7 +12,7 @@
 - AddCoinsCmd
```
Comment (SparklingComet) [inline TODO-Core.txt]: Here, for example, you could add "### " and remove the "-"

**Comment by SparklingComet** [comment]
Also, please replace "-" with "*" (bullet point char)

**Comment by SparklingComet** [comment]
@TyFoxy anything you would like to add?

**Comment by TyFoxy** [comment]
Looks fine so far :D

**Pull request merged**